### ISSUE TITLE
Recorder starts automatically despite manual config

### ISSUE THREAD

**Issue by sheldonbaker** [arut/nginx-rtmp-module#1208]
Starting a stream (from OBS, e.g.) always starts a new .flv file, even though I have `manual` configuration on the recorder.`curl`ing `http://localhost:8080/control/record/stop?app=live&name=test2&rec=default` multiple times also generates *new* recordings.

```
worker_processes 1;
events {
    worker_connections 1024;
}

rtmp {
    server {
        listen 1935;

        application live {
            live on;
            
            recorder default {
                record all manual;
                record_unique on;
                record_path /Users/sheldon/projects/gt/recorder/finished;
            }     
        }
    }
}

http {
    server {
        listen 8080;
        server_name localhost;

        location /control {
            rtmp_control all;
        }

        location /stats {
            rtmp_stat all;
        }
    }
}
```

**Comment by vigikaran** [comment]
yes seems a bug. i tried all the combination of  recorders. no luck

**Comment by gdomod** [comment]
record manual all;

works for me

**Comment by tnakhleh** [comment]
Any update about this issue? I have the same case

**Comment by dogenius01** [comment]
Same issue.

**Comment by sriharshapr** [comment]
Any update on this issue.?

**Comment by kanaldro** [comment]
I confirm the issue.
I installed the [https://github.com/sergey-dryabzhinsky/nginx-rtmp-module/](https://github.com/sergey-dryabzhinsky/nginx-rtmp-module/) branch and the recording starts automatically if it is set to manual.

**Comment by rishpandey** [comment]
Any updates?

**Comment by Alzy** [comment]
I'm also having this issue!!! Any updates on this?

**Comment by Alzy** [comment]
UPDATE! I have confirmed that this does in fact work!!! The https://github.com/sergey-dryabzhinsky/nginx-rtmp-module/ branch that @kanaldro did not work in my tests but I tested with the latest nginx version 1.21.4 and rtmp module version 1.2.2 and it works perfectly.  Set `worker_processes 1;` at the top of the conf and set `record manual all;` for your application.

**Comment by nicoasi** [comment]
> UPDATE! I have confirmed that this does in fact work!!! The https://github.com/sergey-dryabzhinsky/nginx-rtmp-module/ branch that @kanaldro did not work in my tests but I tested with the latest nginx version 1.21.4 and rtmp module version 1.2.2 and it works perfectly. Set `worker_processes 1;` at the top of the conf and set `record manual all;` for your application.

Could you share your conf file because with everything up to date, worker_process 1 and record manual all, recordings still start automatically when stream is on...
Thanks
